class4.html

```diff
@@ -631,19 +645,14 @@ <h2>Questions?</h2>
 					<p>Questions about JSON or Ajax before we wrap up?</p>
 				</section>
 
-				<section>
-					<h3>Survey</h3>
-					<p>Survey info and links in <a href="https://gdidayton.slack.com" target="_blank">slack</a>.</p>
-				</section>
-
 				<section>
 					<h3>Thank you!</h3>
-					<p>As you keep working with JavaScript, if you have questions or get stuck, post in our <a href="https://gdidayton.slack.com">slack</a> and we'll help you out!.</p>
+					<p>As you keep working with JavaScript, if you have questions or get stuck, post in our <a href="https://gdiseattle.slack.com">slack</a> and we'll help you out!.</p>
 				</section>
 			</div>
 			<footer>
 				<div class="copyright">
-					JavaScript 201 -- Dayton --
+					JavaScript 201 -- Seattle --
```

Comment:
Not sure if this matters, but perhaps we should be consistent with the class name. You changed 201 to intermediate JS below, let's just stick with one of those. I think Intermediate JS would be best, so let's change all of these 201's to that.